src/sentry/runner/commands/createuser.py

```diff
@@ -68,23 +80,31 @@ def createuser(email, password, superuser, no_password, no_input, force_update):
         if superuser is None:
             superuser = _get_superuser()
 
+        # for self hosted to give superusers admin permissions
+        if superuser and settings.SENTRY_SELF_HOSTED and superadmin is None:
```

Comment:
How is it possible for `superadmin` to be `None` here in normal usage? The click default is `False`, and specifying the flag explicitly one way or another will likewise result in a boolean.